Tests/HtmlSnapshotTestingTests/__Snapshots__/HtmlSnapshotTestingTests/testSnapshots.1.html

```diff
@@ -1,136 +1,79 @@
 <a>
-  foo1
```

Comment:
are all the deletions in this file intended?

Reply:
I think so but will audit. We were testing each overload before.